janus.c

```diff
@@ -3325,10 +3342,12 @@ gint main(int argc, char *argv[])
 	janus_ice_init(ice_lite, ice_tcp, ipv6, rtp_min_port, rtp_max_port);
 	if(janus_ice_set_stun_server(stun_server, stun_port) < 0) {
 		JANUS_LOG(LOG_FATAL, "Invalid STUN address %s:%u\n", stun_server, stun_port);
+		janus_pidfile_remove();
```

Comment:
That's why I love you guys: you always manage to tell me about things I had no idea existed! :smile: 
I'll update the code to do that, thanks!